src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -67,6 +67,8 @@ public static boolean isNonZeroUnsignedInteger(String s) {
      * Calculates the minimum number of single-character edits (insertions, deletions, or substitutions)
      * needed for the string {@code b} to become a substring of the string {@code a}.
      *
+     * This code was generated with the assistance of GitHub Copilot, an AI-powered code completion tool.
```

Comment:
Could this attribution to AI be updated to follow the conventions?

//@@author {yourGithubUsername}-reused
//{Info about the source...}

{Reused code (possibly with minor modifications) here ...}

//@@author